models/models.go

```diff
@@ -82,6 +82,11 @@ type DataPortalSearchPackage struct {
 	Series []InflatedSeries		`json:"series,omitempty"`
 }
 
+type DataPortalAnalyzerPackage struct {
+	Categories		[]Category				`json:"categories,omitempty"`
+	InflatedSeries	[]InflatedSeries		`json:"series,omitempty"`
```

Comment:
I think the IDE shouldn't like this tab alignment.

Reply:
Funny, it looks ok in the IDE.  I'll see if I can make it right in both places.

Reply:
Apparently just a github display artifact.